psycop/projects/cvd/model_training/preprocessing/datetime_filter.py

```diff
@@ -0,0 +1,17 @@
+import polars.selectors as cs
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("temporal_col_filter")
+class TemporalColumnFilter(PresplitStep):
```

Comment:
Super useful presplitstep -- one vote for common! Had to look `cs.temporal` to understand what a temporal column entails though. Perhaps adding the following as docstring would make it work for common:

"""Removes temporal columns (e.g. of type datetime) from the dataframe"""